app_legacy/Helpers/database/player-game.php

```diff
@@ -436,13 +436,19 @@ function getTotalUniquePlayers(int $gameID, ?string $requestedBy = null, bool $h
         $requestedByStatement = 'OR ua.User = :requestedBy';
     }
 
+    if ($parentGameID !== null) {
+        $bindings['parentGameId'] = $parentGameID;
+    }
+
+    $gameIdStatement = $parentGameID === null ? 'ach.GameID = :gameId' : 'ach.GameID IN (:gameId, :parentGameId)';
```

Comment:
This check is much better than my above suggestion in game.php for `WHERE ach.GameID IN (:gameId , :parentGameId) AND ach.Flags = :achievementType` so it should probably be applied there as well.

Reply:
I've massaged a similar check into _game.php_. Unfortunately, I was not able to do a one-liner due to a potential parameters count mismatch in `$bindings`. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/e5b5a5c1e1bffe9a4fe9f3ff80f811756dc66590.